sources/packages/web/src/components/generic/Banner.vue

```diff
@@ -9,14 +9,14 @@
       <div class="label-bold-normal">{{ header }}</div>
     </template>
     <v-row>
-      <v-col md="10">
+      <v-col md="8">
```

Comment:
I think we can add "auto" for this col. so that, it will auto-adjust for the banners with no btns

Reply:
Thanks @ann-aot auto works without buttons but it will look like this with buttons. So I don't think it will work.
![image](https://user-images.githubusercontent.com/56935198/201758408-63b987eb-4c39-49ee-8174-d784d093c059.png)

Reply:
try to remove "md" from the 2nd "v-col" and add "auto" to first "v-col" and see

Reply:
okay... can you do something like this and see 
![image](https://user-images.githubusercontent.com/77353155/201788956-a5a8288f-1367-4d7a-a171-bfa0e04f66e2.png)

Reply:
![image](https://user-images.githubusercontent.com/77353155/201789015-1ae27b71-bee8-4943-a241-1bc8d38b2752.png)
![image](https://user-images.githubusercontent.com/77353155/201789034-447f7a2b-9ff1-4c36-8978-258233eb706b.png)
**zoom:150%**
![image](https://user-images.githubusercontent.com/77353155/201789119-ccec1980-6dfb-48cb-b412-b521de558c32.png)

Reply:
Thanks @ann-aot Hmm I like auto. But should consider the banner incase we have more text. The shot below shows a banner (button) with more text.
![image](https://user-images.githubusercontent.com/56935198/201795059-82162dbc-0c6a-4f99-9074-110c7e85f697.png)

Reply:
Just a suggestion.
```html
   <v-row>
      <v-col>
        <div class="label-value-normal">
          <slot name="content">{{ summary }}</slot>
        </div>
      </v-col>
      <v-col cols="auto"> <slot name="actions"></slot></v-col>
    </v-row>
```

Reply:
Thanks @andrewsignori-aot that works.

With button
![image](https://user-images.githubusercontent.com/56935198/202308924-9bbc063a-e57b-4081-b7c4-c26b3c59419d.png)

Without button
![image](https://user-images.githubusercontent.com/56935198/202309044-bab992a9-395a-43a3-b341-a86616f4b5ee.png)